docs/layers/ecs/tutorials/1password-scim-bridge.mdx

```diff
@@ -29,14 +29,14 @@ The implementation of this is fairly simple. We will generate credentials for th
       1. Click Integrations in the sidebar
       1. Select "Set up user provisioning"
       1. Choose "Custom"
-      1. You should now see the SCIM bridge credentials. We will need the "Bearer Token" for the next steps.
-      1. Save this token in a secure location (such as 1Password) for future reference
-      1. Store the credentials in AWS SSM Parameter Store. This will allow the ECS task definition to access the credentials securely.
+      1. You should now see the SCIM bridge credentials. We will need the "scimsession" and "Bearer Token" for the next steps.
+      1. Save these credentials in a secure location (such as 1Password) for future reference
+      1. Store the "scimsession" in AWS SSM Parameter Store. This will allow the ECS task definition to access the credentials securely. Then the server will ask for the bearer token using the session, which we will enter at that time.
```

Comment:
```suggestion
      1. You should now see the SCIM bridge credentials. We will need the "scimsession" and "Bearer Token" for the next steps.
      1. Save these credentials in a secure location (such as 1Password) for future reference
      1. Store only the "scimsession" in AWS SSM Parameter Store. This will allow the ECS task definition to access the credentials securely. Then once the service is running, the server will ask for the bearer token to verify the connection, which we will enter at that time.
```